var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -53,14 +56,21 @@ class PyMdanalysis(PythonPackage):
     depends_on("py-threadpoolctl", type=("build", "run"))
     depends_on("py-packaging", type=("build", "run"))
     depends_on("py-fasteners", type=("build", "run"))
-    depends_on("py-gsd@1.9.3:", type=("build", "run"))
+    depends_on("py-gsd@1.9.3:", when="@:2.5.0", type=("build", "run"))
 
     # extra_format (extras_require)
     depends_on("py-netcdf4@1.0:", when="+extra_formats", type=("build", "run"))
     depends_on("py-h5py@2.10:", when="+extra_formats", type=("build", "run"))
+    depends_on("py-pytng@0.2.3:", when="+extra_formats", type=("build", "run"))
     depends_on("py-chemfiles@0.10:", when="+extra_formats", type=("build", "run"))
     depends_on("py-pyedr@0.7.0:", when="+extra_formats", type=("build", "run"))
-    depends_on("py-pytng@0.2.3:", when="+extra_formats", type=("build", "run"))
+    #    depends_on("py-gsd@3.0.1:", when="+extra_formats @2.6.0:", type=("build", "run"))
+    depends_on(
+        "rdkit@2020.03.1: +python ~coordgen ~maeparser ~yaehmop ~descriptors3d",
+        when="+extra_formats @2.6.0:",
+        type=("build", "run"),
+    )
+    depends_on("py-parmed", when="+extra_formats @2.6.0:", type=("build", "run"))
```

Comment:
Can you keep these in the same order as they appear in pyproject.toml? Makes it easier to review

Reply:
Sorry, I was following the order in `setup.py` (which is not consistent with the order in `pyproject.toml`)

Reply:
Do you want me to reorder according to the pyproject.toml anyways?